client-mvp-04/package.json

```diff
@@ -37,7 +37,9 @@
     "build": "react-scripts build",
     "test": "react-scripts test --watchAll=false",
     "test:watch": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint './**/*.{js,jsx}'",
```

Comment:
This pattern doesn't work on my Windows laptop. The script `npm run lint` fails with the following error:
```
No files matching the pattern "'./**/*.{js,jsx}'" were found.
Please check for typing mistakes in the pattern.
```
I tried the following pattern:
```
"lint": "eslint \"**/*.{js,jsx}\""
```
It works on my machine. Please, check it out on macOS.

Reply:
Okay, I will change it. Yes, it works on mack also Josh had used that. I changed that to a single quotation because of readability.